packages/components/stories/icon/icons.js

```diff
@@ -406,6 +406,7 @@ export const icons =
         'IcEmptyStar',
         'IcEwallet',
         'IcEye',
+        'IcFacebook',
```

Comment:
Do we need to include icons file in this PR ?

Reply:
X